backend/.env.sample

```diff
@@ -6,6 +6,10 @@ IS_JOBS_INSTANCE=true
 BACKEND_URL=localhost:4000/backend
 RAILS_RELATIVE_URL_ROOT=/backend
 
+GCP_REGION=
+GCP_PROJECT_ID=
```

Comment:
Hmm, I see we need to set that anyway for cloudtasker?

Reply:
Tiago thinks we don't, that was my first attempt 😊

Reply:
that depends probably on this gem, should it also use some default region etc.

Reply:
and looks like project_id needs to be set in  cloudtasker initializer. But it could probably be taken from `Google::Auth.get_application_default`